src/windows/views/properties_tableview.py

```diff
@@ -577,21 +754,34 @@ def contextMenuEvent(self, event):
                 # Divide into smaller QMenus (since large lists cover the entire screen)
                 # For example: Transitions -> 1 -> sub items
                 SubMenu = None
-                SubMenuRoot = menu.addMenu(_(choice["name"]))
+                if choice["icon"] is not None:
+                    SubMenuRoot = menu.addMenu(choice["icon"], _(choice["name"]))
```

Comment:
I don't think we need the _() translation function here. You have already added translated strings to the self.choices list, right?